config/flatcar/v1_2_exp/translate.go

```diff
@@ -22,15 +22,9 @@ import (
 	"github.com/coreos/vcontext/report"
 )
 
-var (
-	fieldFilters = cutil.NewFilters(types.Config{}, cutil.FilterMap{
-		"storage.luks.clevis": common.ErrClevisSupport,
-	})
-)
-
 // Return FieldFilters for this spec.
 func (c Config) FieldFilters() *cutil.FieldFilters {
-	return &fieldFilters
+	return &cutil.FieldFilters{}
```

Comment:
```suggestion
	return nil
```
Thanks for the PR! The only thing I would see is that the other `translate.go` files use `nil` here

Reply:
Changed it to `nil` here. (This also required a tiny change to the test logic, handling the case of `FieldFilters()` returning `nil`.)